bun-tests/fake-snippets-api/fixtures/get-test-server.ts

```diff
@@ -13,7 +15,7 @@ interface TestFixture {
 }
 
 export const getTestServer = async (): Promise<TestFixture> => {
-  const port = 3001 + Math.floor(Math.random() * 999)
+  const port = 3001 + testNumber++
```

Comment:
can you use `get-port`/ `getPort()` instead?

Reply:
honestly this isn't a bad approach too, maybe combine them- i think `getPort` allows you to specify a preferential port

Reply:
@seveibar the random calculation as before but we pass it to getPort and it checks whether the port is used or not?

Reply:
no sorry, your `testNumber++` approach passed to `getPort(...)` is my rec